README.md

```
@@ -12,29 +12,32 @@
+## OPNSense Firmware Requirements
 
-A Discord server to discuss the integration is available, please click the Discord badge at the beginning of the page for the invite link.
+* Requires OPNsense Firmware 25.1+
+* For OPNsense Firmware 25.1 through 26.1.0, the integration remains supported, but Firewall and NAT rule switches are not available.
+* With OPNsense Firmware 26.1.1+, Firewall and NAT rule switches are available as well.
 
 ## Table of Contents
 
 * [Installation](#installation)
-  * [OPNsense Plugin (deprecated)](#opnsense-plugin)
   * [Home Assistant Integration](#homeassistant-integration)
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)